proposals/templates/proposals/proposal_list.html

```diff
@@ -118,6 +124,8 @@ <h3>{% trans "Uitleg" %}</h3>
                             <img src="{{ img_hide }}" title="{% trans 'Verberg' %}">
                         </a>
                     {% endif %}
+                    {% if proposal.ALLOW_DETAIL_VIEW %}

```

Comment:
Waar is dit voor?

Kan dit weg?

Reply:
Onbedoeld restant van toen ik overwoog de detailpagina zichtbaar te maken voor supervisors. Is nu weg.